ui/js/services/main.js

```diff
@@ -21,6 +21,12 @@ treeherder.factory('thUrl', [
             },
             getLogViewerUrl: function(job_id) {
                 return "logviewer.html#?job_id=" + job_id + "&repo=" + $rootScope.repoName;
+            },
+            getBugIdUrl:function(bug_id) {
+                return "https://bugzilla.mozilla.org/show_bug.cgi?id="+bug_id;
+            },
+            getMachineNameUrl:function(machine_name) {
```

Comment:
I think maybe the function name might be better as `getSlaveHealthUrl` rather than `getMachineNameUrl`?